dockers/docker-database/database_config.json.j2

```diff
@@ -2,7 +2,7 @@
     "INSTANCES": {
         "redis":{
             "hostname" : "{{HOST_IP}}",
-            "port" : 6379,
+            "port" : {{REDIS_PORT}},
             "unix_socket_path" : "/var/run/redis{{NAMESPACE_ID}}/redis.sock",
```

Comment:
DPU should use different unix socket path.

Reply:
Done